readmes/README.Linux.md

```diff
@@ -1,108 +1,173 @@
-## For CentOS:
+# Set up development environment for Cloudberry Database on Linux
 
-- Install Dependencies
+This document shares how to set up the development environments on different Linux systems (CentOS 7, RHEL, and Ubuntu). Before you start to [explore the Cloudberry Database](https://github.com/cloudberrydb/cloudberrydb/tree/main/readmes), follow this document to download and install the required tools and dependencies, and create demo user.
 
-  ```bash
-  ./README.CentOS.bash
-  ```
+Take the following steps to set up the development environments:
 
-- If you want to link cmake3 to cmake, run:
+1. [Clone GitHub repo](#step-1-clone-github-repo).
+2. [Install dependencies](#step-2-install-dependencies).
+3. [Perform prerequisite platform tasks](#step-3-prerequisite-platform-tasks).
 
-  ```bash
-  sudo ln -sf /usr/bin/cmake3 /usr/local/bin/cmake
-  ```
+## Step 1. Clone GitHub repo
 
-- Make sure that you add `/usr/local/lib` and `/usr/local/lib64` to
-`/etc/ld.so.conf`, then run command `ldconfig`.
+Clone the GitHub repository `cloudberrydb/cloudberrydb` to the target machine using `git clone git@github.com:cloudberrydb/cloudberrydb.git`.
 
-- If you want to install and use gcc-7 by default, run:
+## Step 2. Install dependencies
 
-  ```bash
-  sudo yum install -y centos-release-scl
-  sudo yum install -y devtoolset-7-toolchain
-  echo 'source scl_source enable devtoolset-7' >> ~/.bashrc
-  ```
+Install dependencies according to your operating systems:
 
-## For RHEL:
+- [For CentOS 7](#For-CentOS-7)
+- [For RHEL 7 and 8](#For-RHEL-7-and-8)
+- [For Ubuntu 18.04 or later](#For-Ubuntu-1804-or-later)
```

Comment:
```suggestion
- [For CentOS 7](#for-centos-7)
- [For RHEL 7 and 8](#for-rhel-7-and-8)
- [For Ubuntu 18.04 or later](#for-ubuntu-1804-or-later)
```

Reply:
Okay, I will convert the letters to lowercase as required